src/svc.c

```diff
@@ -230,39 +230,13 @@ svc_init(svc_init_params *params)
 	return true;
 }
 
-struct rpc_msg *
-alloc_rpc_msg(void)
-{
-	struct rpc_msg *msg = mem_alloc(sizeof(struct rpc_msg));
-
-	TAILQ_INIT_ENTRY(msg, msg_q);
```

Comment:
Ah. Now copied. No alloc/free.

-		free_rpc_msg(ctx->msg);	/* free call header */
-		ctx->msg = msg;	/* and stash reply header */
+		ctx->cc_msg = *msg;	/* and stash reply header */